### DISCUSSION TITLE
fastapi-cli has not included in fastapi[standart] == 0.112.1 (latest) - request to update documentation

### DISCUSSION THREAD

**Discussion by Bazulenkov** [fastapi/fastapi#12053] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
pip install fastapi[standart]
...
fastapi dev main.py

>> fastapi: command not found
```

### Description

I read on [site](https://fastapi.tiangolo.com/#installation) and it github [here](https://github.com/fastapi/fastapi#installation) also that  after I install `fastapi[standart]` I can use `fastapi dev <some>.py`

But now I see fastapi-cli has not included in fastapi[standart] and I shoud to install it manually for example like this: `pip install fastapi-cli`.

I  request to update documentation or include fastapi-cli to fastapi [standart].

### Operating System

Linux

### FastAPI Version

0.112.1

### Pydantic Version

2.8.2

### Python Version

Python 3.12.3

**Comment by AIdjis** [comment, marked as answer]
Your command is incorrect. It should be [standard], not [standart]. There's a spelling error. If this mistake occurs, it will only install FastAPI directly.
* Use this command to install all the dependencies, including fastapi-cli:
```console
pip install "fastapi[standard]"
```

**Comment by ttnick** [comment]
I am experiencing a similar problem (with standard spelled correctly).
I created a vanill `venv` using Python 3.11:
```console
$ python3.11 -m venv .venv 
$ source .venv/bin/activate
$ pip install "fastapi[standard]"
Collecting fastapi[standard]
  Using cached fastapi-0.53.2-py3-none-any.whl (48 kB)
WARNING: fastapi 0.53.2 does not provide the extra 'standard'
Collecting starlette==0.13.2
  Using cached starlette-0.13.2-py3-none-any.whl (59 kB)
Collecting pydantic<2.0.0,>=0.32.2
  Using cached pydantic-1.7.4-py3-none-any.whl (107 kB)
Installing collected packages: starlette, pydantic, fastapi
Successfully installed fastapi-0.53.2 pydantic-1.7.4 starlette-0.13.2
```
It is already concerning to me that I get `fastapi-0.53.2` and not the latest (`fastapi-0.136.3`); according to PyPI it should be compatible with Python 3.11. I believe the `[standard]` option was added later than 0.53.2, hence the warning.
This also does not install the `fastapi-cli`:
```console
$ fastapi dev
zsh: command not found: fastapi
```

**Reply by YuriiMotov** [reply]
You need to upgrade FastAPI to at least 0.112.0 to make `pip install "fastapi[standard]"` work

**Reply by ttnick** [reply]
That's the problem: `pip` only finds versions 0.1.0 to 0.53.2 for me. If I try to force it, I get:
```console
$ pip install fastapi==0.136.3
ERROR: Could not find a version that satisfies the requirement fastapi==0.136.3 (from versions: 0.1.0, 0.1.2, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.1.10, 0.1.11, 0.1.12, 0.1.13, 0.1.14, 0.1.15, 0.1.16, 0.1.17, 0.1.18, 0.1.19, 0.2.0, 0.2.1, 0.3.0, 0.4.0, 0.5.0, 0.5.1, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.7.0, 0.7.1, 0.8.0, 0.9.0, 0.9.1, 0.10.0, 0.10.1, 0.10.2, 0.11.0, 0.12.0, 0.12.1, 0.13.0, 0.14.0, 0.15.0, 0.16.0, 0.17.0, 0.18.0, 0.19.0, 0.20.0, 0.20.1, 0.21.0, 0.22.0, 0.23.0, 0.24.0, 0.25.0, 0.26.0, 0.27.0, 0.27.1, 0.27.2, 0.28.0, 0.29.0, 0.29.1, 0.30.0, 0.30.1, 0.31.0, 0.32.0, 0.33.0, 0.34.0, 0.35.0, 0.36.0, 0.37.0, 0.38.0, 0.38.1, 0.39.0, 0.40.0, 0.41.0, 0.42.0, 0.43.0, 0.44.0, 0.44.1, 0.45.0, 0.46.0, 0.47.0, 0.47.1, 0.48.0, 0.49.0, 0.49.1, 0.49.2, 0.50.0, 0.51.0, 0.52.0, 0.53.0, 0.53.1, 0.53.2)
ERROR: No matching distribution found for fastapi==0.136.3
```
Recall that I am using Python 3.11 in a perfectly vanilla virtual environment. 
I am sure I am missing something easy here.

**Reply by YuriiMotov** [reply]
Looks like your pip is using non-standard index\mirror..
Try upgrading pip to the latest version and running `pip install fastapi==0.136.3 -i https://pypi.org/simple`

If it doesn't help, please open new discussion and let's continue there

**Reply by ttnick** [reply]
This seems to be the root of the problem. I think my Python 3.11 is from `apt` and that kind of fixed `pip` to version 22, whereas Python 3.10 and 3.14 have 26. It is a bit unsettling---I guess I should clean up a bit in my Python installations---but with that pip version, everything works perfectly. Thank you!
